.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -10,14 +10,17 @@ body:
       label: Search before asking
       description: |-
         Please search [issues](https://github.com/FasterXML/jackson-module-kotlin/issues) to check if your issue has already been reported.
+        Check [all value class labeled issues](https://github.com/FasterXML/jackson-module-kotlin/issues?q=is%3Aopen+is%3Aissue+label%3A%22value+class%22), especially if the problem is related to a value class.
         Also, KotlinModule is only an extension of databind, so it is not an appropriate place to report problems with databind or other modules.
         Please try to ensure that the problem occurs only in Kotlin, and not regular Java objects.
       options:
         - label: "I searched in the [issues](https://github.com/FasterXML/jackson-module-kotlin/issues) and found nothing similar."
           required: true
+        - label: "I have confirmed that the same problem is not reproduced if I exclude the KotlinModule."
+          required: true
```

Comment:
If the problem is not related to `KotlinModule`, it can often be reproduced without `KotlinModule`, so I have added a procedure.